fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -86,6 +88,12 @@
     public static final Predicate<String> isManagedNamespace = p -> p.equals(REPOSITORY_NAMESPACE) ||
             p.equals(LDP_NAMESPACE) || p.equals(MEMENTO_NAMESPACE);
 
+    /**
+     * Tests if we are trying to set the rdf:type to an object with a restricted namespace.
```

Comment:
Maybe update this comment to describe what qualifies as a "restricted namespace", vs talking about "setting" rdf:types... which this predicate is not doing.